include/readout/ReadoutTypes.hpp

```diff
@@ -21,6 +23,7 @@ namespace readout {
 namespace types {
 
 using cfg_data_t = nlohmann::json;
+typedef uint32_t word_t;
```

Comment:
Please "typedef" or "using" right before you need it. I would rename the type to: tpword_t; (I guess you are only using this for those types?)